### ISSUE TITLE
[Feature request] Allow split-screen mode

### ISSUE THREAD

**Issue by carnegie-jwilliamson** [darshanparajuli/SimpleSystemMonitorApp#7]
When monitoring how much CPU a process is using, it would be valuable to see the graphs at the same time the interaction is happening. Would allowing split-screen mode be feasible?

**Comment by darshanparajuli** [comment]
Yes it certainly is. I turned it off at the time because, even though it wasn't crashing, some of the UI layout was getting messed up. I was too busy to properly fix it, and since then, I had forgotten about it. I'll try to properly fix the layout and enable split mode on the next update. Thanks!

**Comment by carnegie-jwilliamson** [comment]
Thanks!
